Library/Formula/open-scene-graph.rb

```diff
@@ -43,7 +45,7 @@ class OpenSceneGraph < Formula
   end
 
   def install
-    ENV.cxx11 if build.cxx11?
+    ENV.cxx11
```

Comment:
Fixed it to remove `option :cxx11`. (Could do a `deprecated_option` instead, but I'm not sure what form it would take for things like `:cxx11`.)